RevenueCatUI/CustomerCenter/Data/Transaction.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Transaction.swift
+//
+//  Created by Facundo Menzella on 5/5/25.
+
+import Foundation
+import RevenueCat
+
+protocol Transaction {
+
+    var productIdentifier: String { get }
+    var store: Store { get }
+    var type: TransactionType { get }
+    var isCancelled: Bool { get }
```

Comment:
what happens with non-cancellable things like consumables / non-consumables?

Reply:
Those return `false`. Let me move the extension conformance code to this file